bill/invoice_test.go

```diff
@@ -685,10 +685,9 @@ func TestRemoveIncludedTaxDeep(t *testing.T) {
 		t.Logf("TOTALS: %v", string(data))
 	*/
 
+	assert.Equal(t, "17830.19", i.Totals.Total.String())
 	assert.Equal(t, "17830.20", i2.Totals.Total.String())
-	// assert.Equal(t, i.Totals.Total.String(), i2.Totals.Total.String())
-	assert.Equal(t, "17830.20", i2.Totals.Total.String())
-	assert.Equal(t, "-0.02", i2.Totals.Rounding.String())
+	assert.Equal(t, "-0.01", i2.Totals.Rounding.String())
```

Comment:
Not sure why this is happening, but it is an improvement.

Reply:
It shouldn't... the PreciseSum method wasn't actually doing anything since the last set of changes.